src/pages/en/core-concepts/astro-pages.md

```diff
@@ -145,6 +150,11 @@ export const get: APIRoute = ({ params, request }) => {
 
 ```
 
+## HTML Pages
+
+You can place a `.html` file in the `/src/pages/` directory. This will be treated as an [HTML Component](/en/core-concepts/astro-components/#html-components) and turned into a page.
```

Comment:
Do we need a "provided it is a full HTML document" kind of clarification here?

Reply:
Are you saying you'd like to come up with one phrase, somewhere between "nothing" and the "existing technical-sounding phrase" and use it in both places? I think that would be fine.

OR if this will still make a page even if it's not a full html page, I'm OK to just leave as is!

Reply:
Turns out this is just out of date. In my testing, if you don't add `<html>` in either an Astro page or an `html` page, it will get added for you, just like `DOCTYPE`.

Reply:
OK, then as agreed, let's leave this as is!